recipes/mosdepth/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.3.8" %}
+{% set version = "0.3.10" %}
```

Comment:
Weird! This recipe misses `source:` data. I wonder how it fetches the sources to build ...

Reply:
Not sure I understand the question, but in `build.sh` it just curls the binary from the github release page